hphp/runtime/vm/jit/vasm-instr.h

```diff
@@ -1109,6 +1110,7 @@ struct shlq { Vreg64 s, d; VregSF sf; Vflags fl; }; // uses rcx
 /*
  * arm intrinsics.
  */
+struct andzi { Immed64 s0; Vreg64 s1, d; VregSF sf; Vflags fl; };
```

Comment:
@mxw - Hi Max,  The intent was to pick something  not 'q'.  The current
naming scheme seems to be for encoding the size of the result.  This is the
second Vinstr to use Immed64.  Do you have any suggestion?